web/components/issues/issue-layouts/list/block.tsx

```diff
@@ -119,33 +129,76 @@ export const IssueBlock = observer((props: IssueBlockProps) => {
       ref={issueRef}
       className={cn(
-        "group min-h-11 relative flex flex-col md:flex-row md:items-center gap-3 bg-custom-background-100 pl-1.5 pr-1 text-sm",
+        "group/list-block min-h-11 relative flex flex-col md:flex-row md:items-center gap-3 bg-custom-background-100 hover:bg-custom-background-90 p-3 pl-1.5 text-sm transition-colors border border-transparent",
         {
-          "border border-custom-primary-70 hover:border-custom-primary-70":
-            getIsIssuePeeked(issue.id) && peekIssue?.nestingLevel === nestingLevel,
-          "last:border-b-transparent": !getIsIssuePeeked(issue.id),
+          "border-custom-primary-70": getIsIssuePeeked(issue.id) && peekIssue?.nestingLevel === nestingLevel,
+          "border-custom-border-400": isIssueActive,
+          "last:border-b-transparent": !getIsIssuePeeked(issue.id) && !isIssueActive,
+          "bg-custom-primary-100/5 hover:bg-custom-primary-100/10": isIssueSelected,
           "bg-custom-background-80": isCurrentBlockDragging,
         }
       )}
     >
-      <div className="flex w-full truncate" style={nestingLevel !== 0 ? { paddingLeft } : {}}>
+      <div className="flex w-full truncate">
         <div className="flex flex-grow items-center gap-3 truncate">
           <div className="flex items-center gap-0.5">
-            <div className="flex items-center group">
-              <DragHandle isDragging={isCurrentBlockDragging} ref={dragHandleRef} disabled={!canDrag} />
-              <div className="flex h-5 w-5 items-center justify-center">
-                {subIssuesCount > 0 && (
-                  <button
-                    className="flex items-center justify-center h-5 w-5 cursor-pointer rounded-sm text-custom-text-400  hover:text-custom-text-300"
-                    onClick={handleToggleExpand}
-                  >
-                    <ChevronRight className={`h-4 w-4 ${isExpanded ? "rotate-90" : ""}`} />
-                  </button>
-                )}
-              </div>
+            {/* drag handle */}
+            <div className="size-4 flex items-center group/drag-handle">
+              <DragHandle
+                ref={dragHandleRef}
+                disabled={!canDrag}
+                className={cn("opacity-0 group-hover/drag-handle:opacity-100", {
+                  "opacity-100": isCurrentBlockDragging,
+                })}
+              />
+            </div>
+            {/* select checkbox */}
+            {projectId && canEditIssueProperties && (
+              <Tooltip
+                tooltipContent={
+                  <>
+                    Only issues within the current
+                    <br />
+                    project can be selected.
+                  </>
+                }
+                disabled={issue.project_id === projectId}
+              >
+                <div className="flex-shrink-0 grid place-items-center w-3.5">
+                  <MultipleSelectEntityAction
+                    className={cn(
```

Comment:
Are these classes different for different views? If not, can you move the classes to the component.

Reply:
The class names depend on the group class, keeping it inside the component might cause confusion. Any better approach for this?